src/Asset/SeatBlack.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="100" height="80" viewBox="0 0 100 80" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Use svg as react component and pass props to change the state of the seat colors.